test/data/stages/zstd/b.mpp.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": "2",
+  "sources": {
+    "org.osbuild.inline": {}
+  },
+  "pipelines": [
+    {
+      "mpp-import-pipeline": {
+        "path": "../manifests/f34-build-v2.json",
+        "id": "build"
+      },
+      "runner": "org.osbuild.fedora34"
+    },
+    {
+      "name": "tree",
+      "build": "name:build",
+      "stages": [
+        {
+          "name": "org.osbuild.rpm",
```

Comment:
I suspect this is what's causing the stage test to fail, this is a v2 API so it should be named "type". I think :) I'm not super familiar with how to generate the osbuild test json.